addons/mail/models/mail_activity.py

```diff
@@ -55,27 +55,25 @@ def default_get(self, fields):
         ('warning', 'Alert'),
         ('danger', 'Error')], string="Decoration Type",
         help="Change the background color of the related activities of this type.")
-    res_model_id = fields.Many2one(
-        'ir.model', 'Model', index=True,
-        domain=['&', ('is_mail_thread', '=', True), ('transient', '=', False)],
+    res_model = fields.Selection(selection=_get_model_selection, string="Model", index=True,
```

Comment:
Actually do we really need that index on res_model ?

Reply:
We often filter on this field but on the other hand, we shouldn't have many records